pom.xml

```diff
@@ -13,13 +13,26 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-compiler-plugin</artifactId>
                 <configuration>
-                    <source>1.7</source>
-                    <target>1.7</target>
+                    <source>8</source>
+                    <target>8</target>
                 </configuration>
             </plugin>
         </plugins>
     </build>
 
```

Comment:
let's have `<properties>` section, which could contain all the artifact versions. example
```
<properties>
       <junit.bom.version>5.8.2</junit.bom.version>
</properties>
```
usage
```
<dependency>
                 <groupId>org.junit</groupId>
                 <artifactId>junit-bom</artifactId>
                 <version>${junit.bom.version}</version>
                 <type>pom</type>
                 <scope>import</scope>
             </dependency>
```
reason: in future you will have to import 2-3 related artifacts (jaxb f.i.) with the one version